src/views/BulkUpload.vue

```diff
@@ -58,7 +58,7 @@
           </ion-item>
         </ion-list>
 
-        <ion-button :disabled="!content.length" color="medium" @click="save" expand="block">
+        <ion-button :disabled="!content.length || isUploadDisabled" color="medium" @click="save" expand="block">
```

Comment:
Instead of disabling this button we should check why the toast message `Select all the fields to continue` is not disabled when mapping is not correct.